tools/internal/mdocspec/ocspec.go

```diff
@@ -0,0 +1,126 @@
+// Package mdocspec parses yaml OC requirements from functional test READMEs.
+package mdocspec
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"sort"
+
+	"github.com/yuin/goldmark"
+	"golang.org/x/exp/maps"
+	"gopkg.in/yaml.v3"
+
+	ppb "github.com/openconfig/featureprofiles/proto/ocpaths_go_proto"
+	rpb "github.com/openconfig/featureprofiles/proto/ocrpcs_go_proto"
+)
+
+// ErrNotFound indicates the user was not found or is unknown.
+var ErrNotFound = errors.New(`did not detect valid "OpenConfig Path and RPC Coverage" yaml, please see https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md for example`)
```

Comment:
OpenConfig Path and RPC Coverage should probably instead refer to the constant `OCSpecHeading`

Maybe https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md should be in a constant as well?

Reply:
Done.

The template file is harder to make a constant. I think we should strive to not change it.